examples/angular/src/pages/ui/components/authenticator/auth-status/auth-status.component.html

```diff
@@ -0,0 +1,20 @@
+<form (submit)="handleSubmit($event)">
```

Comment:
Should we add a note that this example doesn't use the Authenticator because it's testing headless useage of  auth status?

Reply:
Good point. Do you think it'd make sense to put that in the `.feature` file? That's where most of our example explanations go.